### DISCUSSION TITLE
Fedify 1.3.0

### DISCUSSION THREAD

**Discussion by dahlia** [fedify-dev/fedify#194] [Announcements]
The version 1.3.0 of Fedify, an ActivityPub server framework, released! The key changes include:

* [Instrumented spans for OpenTelemetry] are added. In most cases, it's automatically enabled if your application configured [OpenTelemetry SDK].
* Since Fedify 1.3.0, you can configure different [`MessageQueue`]s for incoming and outgoing activities.
* [Fedify now allows you to manually route an `Activity` to the corresponding inbox listener.][1] It is useful for invoking an inbox listener for an `Activity` enclosed by another `Activity`.
* `Context.sendActivity()` and `InboxContext.forwardActivity()` methods now reject when they fail to enqueue the task.
* Thanks to @robin-maki, [@fedify/fedify/x/sveltekit] module is added for integrating with [SvelteKit] hook.
* Fedify now makes HTTP requests with the proper `User-Agent` header, e.g., `User-Agent: Fedify/1.3.0 (Deno/2.1.2)`.

For details, see the [full changelog] as well!

Fedify 1.3.0 is available at [JSR] and [npm].

[Instrumented spans for OpenTelemetry]: https://fedify.dev/manual/opentelemetry
[OpenTelemetry SDK]: https://opentelemetry.io/docs/languages/js/
[`MessageQueue`]: https://jsr.io/@fedify/fedify@1.3.0/doc/federation/~/MessageQueue
[1]: https://fedify.dev/manual/inbox#manual-routing
[@fedify/fedify/x/sveltekit]: https://fedify.dev/manual/integration#sveltekit
[SvelteKit]: https://kit.svelte.dev/
[full changelog]: https://github.com/dahlia/fedify/releases/tag/1.3.0
[JSR]: https://jsr.io/@fedify/fedify@1.3.0
[npm]: https://www.npmjs.com/package/@fedify/fedify/v/1.3.0

**Comment by ThisIsMissEm** [comment]
> Fedify now makes HTTP requests with the proper User-Agent header, e.g., `User-Agent: Fedify/1.3.0 (Deno/2.1.2)`

This should probably include the Fedify server URL: 
```
Fedify/1.3.0 (Deno/2.1.2; +https://example.social/)
```

**Comment by dahlia** [comment]
@ThisIsMissEm Thanks for pointing out!  We'll fix it in the next minor release.
